src/main/java/com/minecolonies/coremod/colony/jobs/AbstractJob.java

```diff
@@ -330,8 +330,14 @@ public void addTasks(@NotNull final EntityAITasks tasks)
     /**
      * Generate your AI class to register.
      *
+     * Suppressing Sonar Rule squid:S1452
+     * This rule does "Generic wildcard types should not be used in return parameters"
+     * But in this case the rule does not apply because
+     * We are fine with all AbstractJob implementations and need generics only for java
+     *
      * @return your personal AI instance.
      */
+    @SuppressWarnings("squid:S1452")
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Suppressing the 'squid:S1452' warning is not allowed [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1309)